src/commands.ts

```diff
@@ -23,6 +27,8 @@ interface BazelQueryQuickPickItem extends QuickPickItem {
     query_item: BazelQueryItem
 }
 
+let bazelDiagnosticsCollection = languages.createDiagnosticCollection("bazel");
+
 function bzlMakeQueryQuickPickItemParsed(queryItem: BazelQueryItem) {
     let {pkg, target} = bzlDecomposeLabel(queryItem.label);
```

Comment:
`bzlDecomposeLabel` now returns also the `workspace` (e.g. remote dependencies) and shows it in the QuickPick.

Reply:
My branch was just out of date. I've merged it in now.